<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="whatGroup">
     <property name="title">
      <string>What ?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="searchInfo">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="replaceInfo">
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="searchEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="replaceEdit"/>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="SearchDrop" name="dropZone">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="whereGroup">
     <property name="title">
      <string>Where ?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="6">
       <widget class="FolderTree" name="directoryTree">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="replaceInFilesCheck">
        <property name="text">
         <string>Replace In Files</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QToolButton" name="removeDirectoryButton">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="addDirectoryButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="executeGroup">
     <property name="title">
      <string>Execute !</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="executeButton">
        <property name="text">
         <string>Rename</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="LogoWidget" name="logoWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LogoWidget</class>
   <extends>QWidget</extends>
   <header>LogoWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FolderTree</class>
   <extends>QTreeWidget</extends>
   <header>FolderTree.h</header>
  </customwidget>
  <customwidget>
   <class>SearchDrop</class>
   <extends>QLabel</extends>
   <header>SearchDrop.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
